midp/background.js

```diff
@@ -91,6 +91,7 @@ Native["com/nokia/mid/s40/bg/BGUtils.maybeWaitUserInteraction.(Ljava/lang/String
 
   // If the page is visible, just start the FG MIDlet
   if (!document.hidden) {
+    profile === 3 && startTimeline();
```

Comment:
> To make it accurate, you could add an asyncImpl with a timeout if profile === 3.

I did that in 16addc27f3338f784d26af4b1a76f8a1fcea791e, but the numbers aren't much closer. Here are some measurements of the time spent in *CldcMIDletSuiteLoader.startSuite* (in ms) under various scenarios:

real warm run: 15239, 13043, 15170
fake warm run: 20478, 19726, 20606
fake warm run with setTimeout: 17258, 17810, 19247